hw/ip/sysrst_ctrl/data/sysrst_ctrl.hjson

```diff
@@ -1,873 +0,0 @@
-// Copyright lowRISC contributors.
```

Comment:
@patel-madhuri should not delete this file, please just revert the change

Reply:
Reverted this file